Dockerfile.ubi

```diff
@@ -9,7 +9,7 @@ FROM quay.io/konveyor/builder:ubi9-v1.23 AS restic-builder
 ENV GOPATH=$APP_ROOT
 RUN mkdir -p $APP_ROOT/src/github.com/restic \
 && cd $APP_ROOT/src/github.com/restic \
-&& git clone https://github.com/konveyor/restic -b oadp-1.3
+&& git clone https://github.com/weshayutin/restic -b 123CVE3
```

Comment:
This is to test ci/prow/images with https://github.com/openshift/restic/pull/38 since the restic repo doesn't do its own image building.

If it passes, close this PR and merge the restic one.